pygmt/tests/test_datasets.py

```diff
@@ -80,9 +80,15 @@ def test_earth_relief_01d():
 
```

Comment:
After bumping to GMT 6.1, we need to add a test with `registration="pixel"` or None.